### PR TITLE
Updated landing page

### PR DESCRIPTION
**Pull request by erwinvanhunen**
Updated landing page to include correct terms and cmdlets to install and update PnP PowerShell

### CODE REVIEW

**Comment by yogkumgit** [comment]
@get-itips

**Comment by get-itips** [comment]
@yogkumgit This one comes from a PnP maintainer so it is ok to merge.

**Review by get-itips — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws @tseward

**Comment by tseward** [comment]
@erwinvanhunen Given we can still use the older 2013-2019 cmdlets and they may be actively used, can we document these 'legacy' cmdets in some other fashion?

Looping in @TroyStarr  as I don't want to see PnP cmdlet support documentation be lost on downlevel versions of SharePoint Server.

File: sharepoint/docs-conceptual/sharepoint-pnp/sharepoint-pnp-cmdlets.md [hunk 1]
```diff
@@ -78,645 +50,16 @@ Connect-PnPOnline -Url https://yoursite.sharepoint.com -UseWebLogin
...
-@SharePointPnPPowerShell.Get-PnPWeb |Returns the current web object|SharePoint Server 2013, SharePoint Server 2016, SharePoint Server 2019, SharePoint Online
-@SharePointPnPPowerShell.New-PnPWeb |Creates a new subweb under the current web|SharePoint Server 2013, SharePoint Server 2016, SharePoint Server 2019, SharePoint Online
-@SharePointPnPPowerShell.Remove-PnPWeb |Removes a subweb in the current web|SharePoint Server 2013, SharePoint Server 2016, SharePoint Server 2019, SharePoint Online
-@SharePointPnPPowerShell.Set-PnPWeb |Sets properties on a web|SharePoint Server 2013, SharePoint Server 2016, SharePoint Server 2019, SharePoint Online
-@SharePointPnPPowerShell.Invoke-PnPWebAction |Executes operations on web, lists and list items.|SharePoint Server 2013, SharePoint Server 2016, SharePoint Server 2019, SharePoint Online
-@SharePointPnPPowerShell.Set-PnPWebPermission |Set permissions|SharePoint Server 2013, SharePoint Server 2016, SharePoint Server 2019, SharePoint Online
-
+### Set and configure credentials ##
+See this [page](https://pnp.github.io/powershell/) for more information on how to use PnP PowerShell and setup various things like the use of a Credential Manager.
```
Comment (JohanFreelancer9) [inline sharepoint/docs-conceptual/sharepoint-pnp/sharepoint-pnp-cmdlets.md:57]:
```suggestion
See this [page](https://pnp.github.io/powershell/) for more information on how to use PnP PowerShell and to set up various things like the use of a Credential Manager.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @erwinvanhunen, kindly commit the minor suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@erwinvanhunen Thanks

**Comment by yogkumgit** [comment]
@erwinvanhunen Gentle reminder

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@dariomws

**Comment by yogkumgit** [comment]
@tiburd

**Pull request merged**